composer/cli/launcher.py

```diff
@@ -485,7 +486,7 @@ def main():
             MOSAICML_PLATFORM_ENV_VAR,
             'false').lower() == 'true' and str(os.environ.get(MOSAICML_LOG_DIR_ENV_VAR, 'false')).lower() != 'false':
         log.info('Logging all GPU ranks to Mosaic Platform.')
-        log_file_format = f'{os.environ.get(MOSAICML_LOG_DIR_ENV_VAR)}/gpu_{{rank}}.txt'
+        log_file_format = f'{os.environ.get(MOSAICML_LOG_DIR_ENV_VAR)}/{os.environ.get(MOSAICML_GPU_LOG_FILE_PREFIX_ENV_VAR)}{{local_rank}}.txt'
```

Comment:
@jjanezhang this will error if the env var doesn't exist, did you want that? or should it be more permissive?